src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControlImpl.cs

```diff
@@ -0,0 +1,491 @@
+                if (_trackInfoChanged == null)
+                {
+                    UnregisterTrackInfoChangedEvent();
+                }
+            }
+        }
+
+        private void RegisterPositionChangedEvent()
+        {
+            _positionChangedCallback = (uint position, IntPtr userData) =>
+            {
+                _positionChanged?.Invoke(null, new PositionChangedEventArgs(position));
+            };
+            int ret = Interop.Bluetooth.SetPositionChangedCallback(_positionChangedCallback, IntPtr.Zero);
+            if (ret != (int)BluetoothError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to set position changed callback, Error - " + (BluetoothError)ret);
+            }
+        }
+
+        private void UnregisterPositionChangedEvent()
+        {
+            int ret = Interop.Bluetooth.UnsetPositionChangedCallback();
+            if (ret != (int)BluetoothError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to unset position changed callback, Error - " + (BluetoothError)ret);
+            }
+        }
+
+        private void RegisterPlayStateChangedEvent()
+        {
+            _playStateChangedCallback = (int state, IntPtr userdata) =>
+            {
+                if (_playStateChanged != null)
+                {
+                    PlayerState State = (PlayerState)state;
+                    _playStateChanged.Invoke(null, new PlayStateChangedEventArgs(State));
+                }
+            };
+            int ret = Interop.Bluetooth.SetPlayStatusChangedCallback(_playStateChangedCallback, IntPtr.Zero);
+            if (ret != (int)BluetoothError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to set play status changed callback, Error - " + (BluetoothError)ret);
+            }
+        }
+
+        private void UnregisterPlayStateChangedEvent()
+        {
+            int ret = Interop.Bluetooth.UnsetPlayStatusChangedCallback();
+            if (ret != (int)BluetoothError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to unset play status changed callback, Error - " + (BluetoothError)ret);
+            }
+        }
+
+        private void RegisterTrackInfoChangedEvent()
+        {
+            _trackInfoChangedCallback = (ref TrackInfoStruct track_info, IntPtr userdata) =>
+            {
+                if (_trackInfoChanged != null)
+                {
+                    Track _track = new Track();
+                    _track.Album = track_info.Album;
+                    _track.Artist = track_info.Artist;
+                    _track.Genre = track_info.Genre;
+                    _track.Title = track_info.Title;
+                    _track.TotalTracks = track_info.total_tracks;
+                    _track.TrackNum = track_info.number;
+                    _track.Duration = track_info.duration;
+                    _trackInfoChanged.Invoke(null, new TrackInfoChangedEventArgs(_track));
+                }
```

Comment:
```suggestion
                  _trackInfoChanged?.Invoke(null, new TrackInfoChangedEventArgs(new Track() {
                      Album = track_info.Album,
                      Artist = track_info.Artist,
                      Genre = track_info.Genre,
                      Title = track_info.Title,
                      TotalTracks = track_info.total_tracks,
                      TrackNum = track_info.number,
                      Duration = track_info.duration
                  }));
```
The underscore(_) should be only used for private members.
And object initializer can be used for new Track.